### DISCUSSION TITLE
Story book with different folder structure

### DISCUSSION THREAD

**Discussion by ajay312** [storybookjs/storybook#17478] [General]
Hi team 
We had different project structure in angular application where storybook is not working .when I added story book to normal angular application.it is working fine. To init sb do we need same angular folder structure?

**Comment by jeremyzilar** [comment]
Storybook can live anywhere in your repo. You will need to use the config to point to the files https://storybook.js.org/docs/react/configure/overview

**Comment by ajay312** [comment]
@jeremyzilar  
![IMG20220210195915](https://user-images.githubusercontent.com/4881209/153428267-79009539-f0d1-4fec-b823-261ab3d4173d.jpg)
Please find attached screenshot and let me know any inputs
